pkg/dashboard/cluster.go

```diff
@@ -510,13 +511,24 @@ func (c *cluster) getResource(ctx context.Context, url, resource string) (*getRe
 	return &res, nil
 }
 
+func addQueryParam(urlStr, key, value string) string {
+	parsedUrl, err := url.Parse(urlStr)
+	if err != nil {
+		return urlStr
+	}
+	query := parsedUrl.Query()
+	query.Add(key, value)
+	parsedUrl.RawQuery = query.Encode()
+	return parsedUrl.String()
+}
+
 func (c *cluster) listResource(ctx context.Context, url, resource string) (listResponse, error) {
 	log.Debugw("list resource in cluster",
 		zap.String("cluster_name", c.name),
 		zap.String("name", resource),
 		zap.String("url", url),
 	)
-
+	url = addQueryParam(url, fmt.Sprintf("labels[%s]", "controller_name"), config.ControllerConfig.ControllerName)
```

Comment:
Please pass it to the dashboard API, they are now coupled.

Reply:
fixed